docs/doxygen/Doxyfile.in

```diff
@@ -507,7 +507,8 @@ EXCLUDE_SYMLINKS       = NO
 
 EXCLUDE_PATTERNS       = */tpl/simpool/* \
                          */tpl/judy/* \
-                         */tpl/conduit/* 
+                         */tpl/conduit/* \
+                         */tpl/cxxopts/* 
```

Comment:
Looks good!

Should we replace all of these with:
```suggestion
                         */tpl/* 
```
That way, we won't have to remember to explicitly add other TPLS (json is coming) to this exclusion list

Reply:
Let's try it and see what happens 😄

Reply:
If this works, I suggest replacing
```
EXCLUDE_PATTERNS  = */tpl/simpool/* \
                        */tpl/judy/* \
                        */tpl/conduit/* \
                        */tpl/cxxopts/*
```
with
```
EXCLUDE_PATTERNS = */tpl/*
```

I wish there was a way I could use the sugguestion button for this...